src/main/java/net/snowflake/client/core/arrow/DateConverter.java

```diff
@@ -21,10 +21,22 @@
 public class DateConverter extends AbstractArrowVectorConverter {
   private DateDayVector dateVector;
   private static TimeZone timeZoneUTC = TimeZone.getTimeZone("UTC");
+  private boolean useDateFormat;
 
   public DateConverter(ValueVector fieldVector, int columnIndex, DataConversionContext context) {
```

Comment:
let's mark this constructor as deprecated and say that the new constructor should be used - we don't use it anymore, but it's part of the public API